src/api/sso/src/authentication.js

```diff
@@ -116,27 +115,29 @@ const strategy = new SamlStrategy(
 
     async function lookupTelescopeUser(id) {
       // We have the user's id (i.e. their hashed email address). Now
-      // make a service-to-service request to the Users service in order to
+      // make a request to the Supabase service in order to
       // get this user's full profile information using the user's id.  They
       // may or may not have a Telescope profile (yet).
       try {
-        const res = await fetch(`${USERS_URL}/${id}`, {
-          headers: {
-            Authorization: `bearer ${createServiceToken()}`,
-          },
-        });
-        if (!res.ok) {
-          if (res.status === 404) {
-            // No Telescope user profile found, so create a regular Seneca user
-            logger.debug({ senecaProfile }, `No Telescope account for ${id} with Users service`);
-            done(null, new User(senecaProfile));
-            return;
-          }
-          // We can't get a response from the Users service, so we don't know what we have.
-          throw new Error(`unable to get user info from Users service: ${res.status}`);
+        const { data: profiles, error } = await supabase
+          .from('telescope_profiles')
+          .select('*')
+          .eq('id', id)
+          .limit(1);
+
+        if (error) {
+          // We can't get a response from the Supabase, so we don't know what we have.
+          throw new Error(`unable to get user info from Supabase: ${error.message}`);
```

Comment:
I'm running the e2e tests locally, and I'm falling into this error constantly.

In the logs for the SSO container, I see:

```
Error: unable to get user info from Supabase: relation "public.telescope_profiles" does not exist
```

It appears that we have a permissions issue.  My limited research into this says that we need to create the `telescope_profiles` table with the `public` schema, so we can access it via the client like this.

From https://www.postgresqltutorial.com/postgresql-schema, it appears that this table should already be in the `public` schema.  Which leads me to think that your tables aren't created, and that you `.sql` file isn't being run.  going to http://localhost:8910/project/default/editor I can see the `auth` schema and tables, but nothing in `public`.

Reply:
I looked inside the `supabase/postgres` container, and I can see the file:

```sh
root@59c20334c37b:/docker-entrypoint-initdb.d# ls -lah
total 32K
drwxr-xr-x 7 root root  224 Feb  9 15:26 .
drwxr-xr-x 1 root root 4.0K Feb  4 23:07 ..
-rw-r--r-- 1 root root 2.5K Feb  2 00:55 00-initial-schema.sql
-rw-r--r-- 1 root root 4.8K Feb  2 00:55 01-auth-schema.sql
-rw-r--r-- 1 root root 4.3K Feb  2 00:55 02-storage-schema.sql
-rw-r--r-- 1 root root 2.8K Feb  2 00:55 03-post-setup.sql
-rw-r--r-- 1 root root  736 Feb  9 15:26 04-initiate-tables.sql
```

So why is it not there?

Reply:
This is weird. I tried running it again locally and I can see `telescope_profiles` and `feeds` tables in public.

![image](https://user-images.githubusercontent.com/51073515/153272029-b1a3b66f-a9d0-4f55-8d6c-a0773495dc9e.png)


<img width="502" alt="Screen Shot 2022-02-09 at 12 15 57 PM" src="https://user-images.githubusercontent.com/51073515/153272261-872b98c6-d54b-4ac3-9812-542c3ab6918e.png">

Reply:
Destroyed and re-created the `supabase-db` container, and I can confirm that it's now running the 4th script and creating these tables:

<img width="951" alt="Screen Shot 2022-02-09 at 2 18 30 PM" src="https://user-images.githubusercontent.com/427398/153274365-810e0433-17d1-44b3-a6a7-929df19a613d.png">

While I was doing this, I noticed that row level security (RLS) is not on for these tables.  In a follow-up, we should think about adding.

Reply:
Yeah, it depends on #1871 to get the JWT payload from requests.